example/android/app/build.gradle

```diff
@@ -192,7 +76,12 @@ android {
 
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
-    implementation "com.facebook.react:react-native:0.68.5"  // From node_modules
+    implementation "com.facebook.react:react-native:0.68.5"
+    implementation "com.yoti.mobile.android.sdk:yoti-sdk-doc-scan:${rootProject.ext.yotiSdkVersion}"
```

Comment:
Before we included all module deps, I'm wondering if we should change that now...

Reply:
But liveness is commented below

Reply:
its not

Reply:
You're right, sorry, scrolled past it